src/openforms/submissions/tests/test_models.py

```diff
@@ -617,3 +620,25 @@ def test_clear_execution_state_without_execution_state(self):
         self.assertFalse(hasattr(submission, "_execution_state"))
 
         submission.clear_execution_state()
+
+
+@temp_private_root()
+class PDFSubmissionReportTests(HypothesisTestCase):
+    @given(
+        st.text(
+            min_size=1,
+            alphabet=st.characters(blacklist_characters="\x00", codec="utf-8"),
+        )
+    )
+    # see ticket #3470 - explicit value for form name (the final file name will be > max_length)
+    # Django's get_available_name truncates original name if required, so with a large
+    # name which has a '/' as well it confuses file_root (which is truncated)
```

Comment:
I don't really understand what this means :thinking: What I understand:
- When creating the `ContentFile`, we now slugify the form name (so the `/` problem is solved). I guess the length of the slugified form name is <= 150 (the max length of the form name).
- On save, the storage class possibly does some additional changes to the filename: https://github.com/django/django/blob/3.2.21/django/core/files/storage.py#L53
But I don't understand what causes the problem, since by now the slashes should have been removed?

Reply:
I added this comment in order to give some content for the specific error. I can remove it if you think it's redundant.

And yes, you are correct, with the slugified version we won't have a problem.

Reply:
No no! It's more that I didn't really understand what the comment meant! :sweat_smile: 
If with the slugified version this is no longer a problem, can we remove this test altogether?

Reply:
Ah no I see! I would then maybe just change the comment to not include the mention to the `/` since that is no longer a problem?